### DISCUSSION TITLE
sussy game

### DISCUSSION THREAD

**Discussion by JackyBorderCollie** [CluckCluckChicken/Magnifier#81] [big chungus]
yessssssssssssss i made a epic spritesheet for the chrome dinosaur and this is what it looks in action it's very sussy:
![](https://u.cubeupload.com/JackyBorderCollie/sjsjsjsjsjjs.png)

**Comment by JackyBorderCollie** [comment]
as u can see iT's vErY sUsSy!1!111!1!!

**Comment by davwat** [comment]
![sus drip](https://user-images.githubusercontent.com/46281840/120468373-0d91be80-c399-11eb-8bd0-cbda51fdb587.png)

**Reply by JackyBorderCollie** [reply]
oh yeeeeeeeeeeeeee amogus dripppppppppppppppp
